### ISSUE TITLE
Hardware references

### ISSUE THREAD

**Issue by m0hai** [DedeHai/ESP-TogglBox#1]
I think some reference to the used hardware could be usefull.
For examle, for each component, his name or his code or/and where to buy it. (in particular oled screen)

**Comment by DedeHai** [comment]
Good point actually. I have a project description at [Hackaday](https://hackaday.io/project/13236-togglcom-project-tracking-box) but even there I do not mention what exact display I used.

**Comment by m0hai** [comment]
Also main board name and with what was flashed.... Simply imagine that someone has to build your project starting from zero...
If you remember them, Ii think that also codes and/or links of bushbuttons and rotary switch can be usefull....
